spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ImportExportControllerHelperService.java

```diff
@@ -503,13 +504,13 @@ public JSONObject confirmLibrariesPoolsUpload(HttpSession session, JSONObject js
                     pools.put(poolName, pool);
                     log.info("Added pool: " + poolName);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)